docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -190,6 +190,22 @@ When diagnosing the health state of the task, you will most likely be interested
 
 Investigating the underlying task can help you gauge whether the problem you’re seeing is rooted in the rule not running at all, whether it’s running and failing, or whether it is running, but exhibiting behavior that is different than what was expected (at which point you should focus on the rule itself, rather than the task).
 
+[discrete]
+[[alerting-kibana-disable]]
+=== Temporarily disable
+
+A cluster may become unresponsive or sluggish if too many or expensive {kib} rules 
+are attempting to run. As a stop gap measure, you may consider temporarily overriding
```

Comment:
> As a stop gap measure, you may consider temporarily overriding

As a novice user, it's unclear to me from this text why/when you'd want to perform these steps as opposed to the steps described in https://www.elastic.co/guide/en/kibana/master/create-and-manage-rules.html#controlling-rules.  Can we provide more context?

Reply:
Agreed, I feel like the framing here is more of a support perspective than that of the user.

Perhaps we can frame it like this (my phrasing is terrible, please do rephrase as needed): 
> If you are experiencing sluggishness in Kibana and wish to switch your rules off temporarily to ensure they are the cause, you can do so like this.

or something along those lines.